dev-python/urwid/urwid-2.1.0.ebuild

```diff
@@ -17,6 +17,10 @@ SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ~ia64 ~mips ppc ppc64 ~sparc x86 ~amd64-linux ~x86-linux"
 IUSE="examples"
 
+PATCHES=(
+	"${FILESDIR}/2.1-Ignore-resetting-to-invalid-locale-Closes-377.patch"
```

Comment:
Don't touch the stable ebuild here at all, I'm not comfortable with it.

Reply:
Ok I am creating a new urwid-2.1.0-r1 with the fix in it